spec/controllers/plugin/plugin_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe PluginController do
+  render_views
+
+  describe '#user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.